src/Controller/DateController.php

```diff
@@ -430,8 +430,8 @@ public function detailAction(
         }
 
         // mark annotations as read
-        $annotationList = $date->getAnnotationList();
-        $annotationService->markAnnotationsReadedAndNoticed($annotationList);
+        //$annotationList = $date->getAnnotationList();
+        //$annotationService->markAnnotationsReadedAndNoticed($annotationList);
```

Comment:
Remove if not necessary

And as in the first comment. DateController might not be the only place where `markAnnotationsReadedAndNoticed()` is called inside a rubric specific controller.

Reply:
@EofChris thank you for comments

Reply:
@EofChris The commenst has been resolved,